recipes/geckodriver/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -ex
+rustc -V
+cargo -V
+
+if [ $(uname) = Darwin ] ; then
+    export CXXFLAGS="-arch $OSX_ARCH -stdlib=libc++ -std=c++11"
```

Comment:
So you may want to ensure that you don't replace conda's `CXXFLAGS` and `CFLAGS`  here for compatibility.   
Does this build also invoke c / c++ somewhere internally?  If so you may want to add this to the meta.yaml

```yaml
  build:
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
```